docs/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## 0.6.0
+
+### New Features
+
+- New config field, "connection_idle_timeout" in Listener, mosn will close idle timeout connection. The default value is 90s. 
```

Comment:
Add "connection_idle_timeout" field to Listener config, default 90s. Mosn will close idle connections when timeout.